_episodes_rmd/04-dplyr.Rmd

```diff
@@ -210,21 +231,21 @@ to a probability value according to the formula:
 
 Probability = 1- 10 ^ -(QUAL/10)
 
-Let's add a column (`POLPROB`) to our `variants` dataframe that shows 
-the probability of a polymorphism at that site given the data. We'll show 
+Let's add a column (`POLPROB`) to our `variants` dataframe that shows
+the probability of a polymorphism at that site given the data. We'll show
 only the first six rows of data.
 
 ```{r, purl = FALSE}
 variants %>%
   mutate(POLPROB = 1 - (10 ^ -(QUAL/10))) %>%
-  glimpse()
+  head()
 ```
 
 > ## Exercise
 > There are a lot of columns in our dataset, so let's just look at the
-> `sample_id`, `POS`, `QUAL`, and `POLPROB` columns for now. Add a 
-> line to the above code to only show those columns. 
-> 
+> `sample_id`, `POS`, `QUAL`, and `POLPROB` columns for now. Add a
+> line to the above code to only show those columns.
+>
 >> ## Solution
```

Comment:
quite a long output

Reply:
new edits were committed addressing this.